CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- Add Android View Hierarchy support ([#2440](https://github.com/getsentry/sentry-java/pull/2440))
```

Comment:
```suggestion
- Attach View Hierarchy to the errored/crashed events ([#2440](https://github.com/getsentry/sentry-java/pull/2440))
```

maybe this gives a little bit more idea what is this all about for those who read